Documentation/ceph-block.md

```diff
@@ -52,11 +52,23 @@ parameters:
     clusterID: rook-ceph
     # Ceph pool into which the RBD image shall be created
     pool: replicapool
+
+    # RBD image format. Defaults to "2".
+    imageFormat: "2"
```

Comment:
@Madhu-1 If it defaults to "2", why do we need it here in the doc? The general approach has been to keep documentation examples as simple as possible by not including defaults.